auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -356,11 +355,33 @@ public void writeToParcel(Parcel parcel, int i) {
             parcel.writeStringList(mScopes);
         }
 
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) return true;
+            if (o == null || getClass() != o.getClass()) return false;
+
+            IdpConfig config = (IdpConfig) o;
+
+            return mProviderId.equals(config.mProviderId);
```

Comment:
@samtstern Identifying providers by provider id only and completely ignoring scopes makes sense to me, do you agree?